app/lib/actions/pulp3/repository/presenters/content_unit_presenter.rb

```diff
@@ -15,6 +15,7 @@ def humanized_details
             ret << _("Total tasks: ") + ": #{finished_units}/#{total_units}"
             ret << "--------------------------------"
             progress_reports = sync_task.try(:[], 'progress_reports') || []
+            progress_reports = progress_reports.sort_by { |pr| pr.try(:[], 'message') }
```

Comment:
Looks good..Even though I am not sure that alphabetically is the natural order for these messages. Maybe in the order they execute but pulp needs to give us that.

Reply:
Yeah, the problem was that the ordering wasn't consistent coming from pulp itself.  But this could be considered a pulp bug, i will chat with them before merging